mltrace/client.py

```diff
@@ -443,13 +433,30 @@ def get_recent_run_ids(limit: int = 5, last_run_id=None):
     return store.get_recent_run_ids(limit, last_run_id)
 
 
+def get_all_run_ids():
+    """Returns most all component run ids."""
+    store = Store(_db_uri)
+    res = store.get_all_run_ids()
+    return [str(x[0]) for x in res]
+
+
 def get_io_pointer(io_pointer_id: str, create=True):
     """Returns IO Pointer metadata."""
     store = Store(_db_uri)
     iop = store.get_io_pointer(io_pointer_id, create)
     return IOPointer.from_dictionary(iop.__dict__)
 
 
+def get_all_tags() -> typing.List[str]:
+    store = Store(_db_uri)
+    res = store.get_all_tags()
+    tags = set()
```

Comment:
Tags are unique so you can keep it a list. See `db/models.py` for the schema

`tags = [t.name for t in res]`

Reply:
Oh nice ok.